dev-docs/bidders/aax.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: bidder
+title: Aax
+description: Prebid Media.net Bidder Adaptor
+biddercode: aax
+gdpr_supported: true  
+media_types: banner,native
+usp_supported: true
+userIds: britepoolId, criteo, id5Id, identityLink, liveIntentId, netId, parrableId, pubCommonId, unifiedId
+prebid_member: false
+pbjs: false
+gvl_id: 142
+schain_supported: true
+floors_supported: true
+fpd_supported: truef
+pbs: true
```

Comment:
Another very useful flag for sales houses / publishers is the `safeframes_ok` . If `true` it means that  your adapter works with safe-frames (none-friendly-iframes).